File: Legacy/Appostazja.Core/Pdf/ApostasyDeclaration.cs
namespace Appostazja.Core.Pdf;

public sealed record ApostasyDeclaration(
    string FullName,
    string HomeAddress,
    DateOnly BaptismDate,
    string BaptismParish,
    string ResidenceParish,
    string Motivation,
    DateOnly DeclarationDate);

File: src/Appostazja.Android/app/src/main/java/com/tryniecki/appostazja/pdf/ApostasyDeclaration.kt
package com.tryniecki.appostazja.pdf

import java.time.LocalDate

data class ApostasyDeclaration(
    val fullName: String,
    val homeAddress: String,
    val baptismDate: LocalDate,
    val baptismParish: String,
    val residenceParish: String,
    val motivation: String,
    val declarationDate: LocalDate,
)
